docs/ops.html

```diff
@@ -3641,7 +3641,6 @@ <h3>Limitations</h3>
       using one of those features, you will not be able to migrate to KRaft yet.</li>
```

Comment:
Maybe we can phrase like this here: `If you are using one of those features, you are not able to migrate to KRaft yet.`